### ISSUE TITLE
type Numeric works incorrectly on v0.8

### ISSUE THREAD

**Issue by thinhbuzz** [elysiajs/elysia#386]
example code:

```typescript
const app = new Elysia()
  .get('/', () => {
    return 'Hello Elysia';
  })
  .post(
    '/demo',
    () => [],
    {
      body: t.Object({
                       limit: t.Number({ minimum: 10, maximum: 1000 }),
                       timestamp: t.Numeric({ minimum: 1672531200 }),
                     }),
    },
  )
  .listen(+(process.env.PORT || 8080));

```

request body:
```json
{
    "limit": 30,
    "timestamp": "1693468537",
    "a": ""
}
```
## 0.7 response
```text
Invalid body, 'a': Unexpected property

Expected: {
  "limit": 10,
  "timestamp": 0
}

Found: {
  "limit": 30,
  "timestamp": "1693468537",
  "a": ""
}
```
## 0.8 response
```text
Object.assign requires that input parameter not be null or undefined
```

## 0.8 response without `timestamp: t.Numeric({ minimum: 1672531200 }),`
```
{
  "type": "body",
  "at": "timestamp",
  "message": "Unexpected property",
  "expected": {
    "limit": 10
  },
  "found": {
    "limit": 30,
    "timestamp": "1693468537",
    "a": ""
  },
  "errors": [
    {
      "type": 42,
      "schema": {
        "type": "object",
        "properties": {
          "limit": {
            "minimum": 10,
            "maximum": 1000,
            "type": "number"
          }
        },
        "required": [
          "limit"
        ],
        "additionalProperties": false
      },
      "path": "/timestamp",
      "value": "1693468537",
      "message": "Unexpected property"
    },
    {
      "type": 42,
      "schema": {
        "type": "object",
        "properties": {
          "limit": {
            "minimum": 10,
            "maximum": 1000,
            "type": "number"
          }
        },
        "required": [
          "limit"
        ],
        "additionalProperties": false
      },
      "path": "/a",
      "value": "",
      "message": "Unexpected property"
    }
  ]
}
```

**Comment by thinhbuzz** [comment]
@SaltyAom any update?

**Comment by MollariDev** [comment]
I believe this was fixed in 0.8.6 as it was the same issue when validating a string with a default value.

**Comment by thinhbuzz** [comment]
@MollariDev it seems so, the owner fixed it silently
